bin/run-browser-tests

```diff
@@ -42,12 +42,20 @@ if [[ -z "$(docker ps --filter "name=${CONTAINER_NAME}" --format "{{.Names}}")"
     civiform-browser-test:latest
 fi
 
+# Detect if we are running tests with an Azure server, by looking for the Azurite
+# image in the docker container list.
+IS_AZURE_SERVER=
+if docker ps --filter "ancestor=mcr.microsoft.com/azure-storage/azurite" -q; then
+  IS_AZURE_SERVER=1
+fi
+
```

Comment:
(blocking) This won't work correctly if both the dev and browser test servers are running azure.

Reply:
removed this part.